src/qibo/quantum_info/entanglement.py

```diff
@@ -280,7 +294,8 @@ def entangling_capability(circuit, samples: int, seed=None, backend=None):
     for _ in range(samples):
         params = local_state.uniform(-np.pi, np.pi, circuit.trainable_gates.nparams)
         circuit.set_parameters(params)
-        entanglement = meyer_wallach_entanglement(circuit, backend=backend)
+        state = backend.execute_circuit(circuit).state()
```

Comment:
This is going to fail when `circuit.density_matrix=False` and are present either noise channels or collapsing measurements, which is ok but maybe we should point it out in the docstring.

Reply:
I fixed the bug for `density_matrix=True` and added a note for channels.